samples/Cancellation/Cancellation.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
@liliankasem would you change this to .NET 7 ? that's why this is why the example will have to stand out from others.  thanks :)

Reply:
Perhaps we can make this sample targeting NET7, similar to the NETFX sample we have.

Reply:
If we're framing is as a .net 7 sample, then I'm okay with the addition of a new sample.